src/main/java/frc/robot/Constants.java

```diff
@@ -43,41 +43,50 @@
     public static final int FRONT_LEFT_MODULE_STEER_MOTOR = 15;
     public static final int FRONT_LEFT_MODULE_STEER_ENCODER = 19;
 
-    //public static final double FRONT_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(139.5278); //practicebot
-    public static final double FRONT_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(142-6); //compbot
+    public static final double FRONT_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(139.5278); //practicebot
+    // public static final double FRONT_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(142-6); //compbot
 
     public static final int FRONT_RIGHT_MODULE_DRIVE_MOTOR = 12;
     public static final int FRONT_RIGHT_MODULE_STEER_MOTOR = 16;
     public static final int FRONT_RIGHT_MODULE_STEER_ENCODER = 20;
 
-   // public static final double FRONT_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(144.2137); //practicebot
-    public static final double FRONT_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(166+180+5); //compbot
+   public static final double FRONT_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(144.2137); //practicebot
+    // public static final double FRONT_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(166+180+5); //compbot
 
     public static final int BACK_LEFT_MODULE_DRIVE_MOTOR = 14;
     public static final int BACK_LEFT_MODULE_STEER_MOTOR = 18;
     public static final int BACK_LEFT_MODULE_STEER_ENCODER = 22;
 
-   // public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(226.3419); //practicebot
-    public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(308); //compbot
+   public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(226.3419); //practicebot
+    // public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(308); //compbot
 
     public static final int BACK_RIGHT_MODULE_DRIVE_MOTOR = 13;
     public static final int BACK_RIGHT_MODULE_STEER_MOTOR = 17;
     public static final int BACK_RIGHT_MODULE_STEER_ENCODER = 21;
 
-    //public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(245.0254); //practicebot
-    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(283+2); //compbot
+    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(245.0254); //practicebot
+    // public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(283+2); //compbot
 
     public static final int SHOOTER_LEFT = 27;
     public static final int SHOOTER_RIGHT = 28;
     public static final double SHOOTER_SPEED = 0.5;
 
     public static final int INDEXER_MOTOR = 24;
-    public static final double FOWARDS_INDEX_SPEED = 0.5;
-    public static final double REVERSE_INDEX_SPEED = -1.0;
+    public static final double FOWARDS_INDEX_SPEED = -0.5;
```

Comment:
My preference would be to use positive values for forward and config the motor to be inverted.

Reply:
I agree, this is a weird case where when we tested, both reverse and forward somehow went forward so I just left the values we used instead of inverting.

I will add a @todo to fix it with more testing.